src/chart-addons/stacked-legend.js

```diff
@@ -238,7 +308,10 @@ export async function getLegendStateFromChart(chart, useMap, selectedLayer) {
                     colValues,
                     color.originalDomain,
                     color.originalRange,
-                    color.defaultOtherRange
+                    color.defaultOtherRange,
+                    color?.fullColorHashing,
```

Comment:
Not sure if there's any utility in the optional chaining here, it will fail before this if color is nullish

Reply:
Since `fullColorHashing` is a new prop added this release, this is to protect against legacy charts not having it in their `color` object

Reply:
The optional chaining here is protecting against `color` being undefined or null. 

If the old charts don't have `fullColorHashing` in the `color` object, this will return `undefined` (as expected I assume) with or without the optional chaining as you have it here.

Reply:
Oh yeah, nevermind, you're right, won't matter since we know at this point `color` is defined